### PR TITLE
Adds support for segmentation settings in Azure STT

### PR DESCRIPTION
**Pull request by nfma**

### CODE REVIEW

**Review by theomonnom — APPROVED** [review]
lgtm! can you sign the CLA?

File: livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/stt.py [hunk 1]
```diff
@@ -43,6 +46,9 @@ def __init__(
         speech_region: str | None = None,
         sample_rate: int = 16000,
         num_channels: int = 1,
+        segmentation_silence_timeout_ms: int = 500,
+        segmentation_max_time_ms: int = 60000,
+        segmentation_strategy: str = "Default",
```
[the remark this answers is not in the document]
Comment (theomonnom) [reply livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/stt.py:51]: Maybe we should default them to None and only use `speech_config.set_properties` when they're set

[the remark this answers is not in the document]
Comment (nfma) [reply livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/stt.py:51]: @theomonnom those values are the default for strategy and silence but not sure for max time - i had to adjust it a bit to make the tests pass.

but i can change it to only set the properties when those values are set and default to None

**Review by theomonnom — APPROVED** [review]
Thanks!

**Pull request merged**